prime-dotnet-webapi/Services/SubmissionService.cs

```diff
@@ -284,6 +297,22 @@ private async Task<bool> AcceptToaAsync(Enrollee enrollee, object additionalPara
                     return false;
                 }
             }
+            else
+            {
+                // for regular enrollee, get the pending agreement, create the PDF and store it in document manager
+                var pendingAgreementId = enrollee.Agreements.OrderByDescending(a => a.CreatedDate).Select(a => a.Id).First();
+                Agreement agreement = await _enrolleeAgreementService.GetEnrolleeAgreementAsync(enrollee.Id, pendingAgreementId, true);
+                var html = await _razorConverterService.RenderTemplateToStringAsync(RazorTemplates.Agreements.PdfNoSignature, agreement);
+                var pdfbinary = _pdfService.Generate(html);
+                var filename = "Terms-Of-Access.pdf";
+                var documentGuid = await _documentManagerClient.SendFileAsync(new System.IO.MemoryStream(pdfbinary), filename, DestinationFolders.SignedOrgAgreements);
```

Comment:
Need a TODO: comment or something because it is somewhat misleading that we are saving enrollee TOAs into the `/app/document_uploads/signed_org_agreements` folder